extensions/amp-carousel/0.1/test/test-slidescroll.js

```diff
@@ -1271,6 +1271,16 @@ describes.realWin(
         expect(showSlideSpy).to.have.been.calledWith(4);
       });
 
+      it('should error on a non-number goToSlide', async () => {
+        const ampSlideScroll = await getAmpSlideScroll(true);
+        const impl = await ampSlideScroll.getImpl();
+        const devErrorSpy = env.sandbox.spy(dev(), 'error');
+
+        impl.showSlideAndTriggerAction_(NaN)
```

Comment:
Might be extra safe to add another unit test that also passes `Infinity` or similar.